src/styles/index.css

```diff
@@ -226,6 +230,71 @@ footer {
   margin: 5px 0;
 }
 
+
+
+.ui.checkbox.major input:focus:checked ~ .box:before,
+.ui.checkbox.major input:focus:checked ~ .coloring.black:before,
+.ui.checkbox.major input:checked ~ .box:before,
+.ui.checkbox.major input:checked ~ .coloring.black:before {
+    background: var(--major-tags) !important;
```

Comment:
uhh so i see u relied a lot on the semantic classes which isnt really convention when using semantic react